src/core/CPU.cpp

```diff
@@ -1194,7 +1190,16 @@ Blaze::Cycles Blaze::CPU::executeEOR(AddressingMode mode) {
 };
 
 Blaze::Cycles Blaze::CPU::executeINC(AddressingMode mode) {
-	// TODO
+	// TODO TO CHECK
+	Address addr = loadOperand(mode);
+	A++;
+	if (memoryAndAccumulatorAre8Bit()) {
+		store8(addr, A.load());
+	}
+	else {
+		store16(addr, A.load());
+	}
```

Comment:
Same comment here as for `DEC`, except it adds 1 instead of subtracting 1.

Reply:
Made the changes.